index.test-d.ts

```diff
@@ -0,0 +1,5 @@
+import {expectType} from 'tsd-check';
+import isFullwidthCodePoint from '.';
+
+const codePoint = '谢'.codePointAt(0) as number;
```

Comment:
I prefer `!` and find it more readable, as it makes it clear it's only about removing `undefined` instead of `as` which could change the type completely.

```suggestion
const codePoint = '谢'.codePointAt(0)!;
```

Reply:
https://github.com/typescript-eslint/typescript-eslint/issues/202